eng/Versions.props

```diff
@@ -73,10 +73,10 @@
     <!-- Several packages from the editor are used for testing HTML support, and share the following version. -->
     <Tooling_HtmlEditorPackageVersion>16.10.57-preview1</Tooling_HtmlEditorPackageVersion>
     <!-- Several packages share the MS.CA.Testing version -->
-    <Tooling_MicrosoftCodeAnalysisTestingVersion>1.0.1-beta1.21103.2</Tooling_MicrosoftCodeAnalysisTestingVersion>
+    <Tooling_MicrosoftCodeAnalysisTestingVersion>1.1.2-beta1.22109.1</Tooling_MicrosoftCodeAnalysisTestingVersion>
     <MicrosoftVisualStudioShellPackagesVersion>17.2.32330.158</MicrosoftVisualStudioShellPackagesVersion>
-    <MicrosoftVisualStudioPackagesVersion>17.3.37-preview</MicrosoftVisualStudioPackagesVersion>
-    <RoslynPackageVersion>4.3.0-3.22303.4</RoslynPackageVersion>
+    <MicrosoftVisualStudioPackagesVersion>17.3.133-preview</MicrosoftVisualStudioPackagesVersion>
+    <RoslynPackageVersion>4.4.0-1.22369.1</RoslynPackageVersion>
```

Comment:
Upgrading Roslyn because we updated our O# version which also uses a newer Roslyn. They are now aligned. Similarly had to update the VS package version because newer Roslyn editor packages depended on newer vs packages